components/list/list-item-drag-mixin.js

```diff
@@ -0,0 +1,114 @@
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { html } from 'lit-element/lit-element.js';
+import { nothing } from 'lit-html';
+
+export const ListItemDragMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			draggable: {type: Boolean, reflect: true },
+			key: { type: String, reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this._itemDragId = getUniqueId();
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		if (this.key === undefined) {
```

Comment:
```suggestion
		if (!this.key) {
```
This should handle all falsey cases to maintain consistency with the `checkbox-mixin`

Reply:
The question though is `0` an acceptable key?